tests/utils/test_scripts.py

```diff
@@ -158,3 +159,72 @@ def test_export_predictions_and_labeled_video(
 
     # clean up logging
     remove_logs()
+
+
+def _supervised_multi_gpu_cfg(cfg):
```

Comment:
this is fine, but you could also make a variable `cfg_multigpu` as a fixture in the file `tests/conf.py` and pass it to whatever test(s) you need

Reply:
Acknowledged. I'm not sure I'll use this is any other test yet.